### PR TITLE
refactor tests to use fixtures

### PR DESCRIPTION
**Pull request by PeterYinusa**
Replaced by #11374

### CODE REVIEW

**Comment by brad-decker** [comment]
needs a rebase, but looks good after that!

File: test/e2e/tests/lock-account.spec.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: 25000000000000000000,
+      },
+    ],
+  };
+  it('accepts the account password after lock', async function () {
```
Comment (adonesky1) [inline test/e2e/tests/lock-account.spec.js:14]: [nit] - maybe "successfully unlocks after lock"

File: test/e2e/tests/add-account.spec.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: 25000000000000000000,
+      },
+    ],
+  };
+  it('should display correct account name', async function () {
```
Comment (adonesky1) [inline test/e2e/tests/add-account.spec.js:14]: [REQ] - maybe 'should display correct new account name after create'

File: .circleci/scripts/chrome-install.sh [hunk 3]
```diff
@@ -6,10 +6,17 @@ set -o pipefail
 
 CHROME_VERSION='79.0.3945.117-1'
 CHROME_BINARY="google-chrome-stable_${CHROME_VERSION}_amd64.deb"
-CHROME_BINARY_URL="http://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/${CHROME_BINARY}"
+CHROME_BINARY_URL="http://mirror.cs.uchicago.edu/google-chrome/pool/main/g/google-chrome-stable/${CHROME_BINARY}"
```
Comment (Gudahtt) [inline .circleci/scripts/chrome-install.sh:9]: This shouldn't be in the diff :thinking: It's already on `develop`.

Comment (PeterYinusa) [reply .circleci/scripts/chrome-install.sh:9]: agreed

**Pull request closed without merging**